precompiles/relay-encoder/RelayEncoder.sol

```diff
@@ -18,12 +18,10 @@ RelayEncoder constant RELAY_ENCODER_CONTRACT = RelayEncoder(
 interface RelayEncoder {
     /// @dev Encode 'bond' relay call
     /// @custom:selector a82948d4
```

Comment:
If we modify the function signature, the associated selector changes as well. The selector is built with the first 4 bytes of the `Keccak-256` hash of the function signature. So in this case, it should be `72a9fbc6`. 

You can use this page to calculate selectors every time you change function signatures in solidity:
https://emn178.github.io/online-tools/keccak_256.html

Just type the function signature (in this case `encodeBond(uint256,bytes)`) and take the first 8 characters (4 bytes) of the resultant hash.